explorer/src/metaplex/Art/Art.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { Ref, useCallback, useEffect, useState } from "react";
+import { Image } from "antd";
```

Comment:
I'm worried that this will bloat the explorer a fair by pulling in this full UI framework. Here are the results of building the explorer with your changes:
```
  906.32 KB (+78.38 KB)  build/static/js/2.6bb9aa5c.chunk.js
  71.52 KB (+5.01 KB)    build/static/js/main.a81573b6.chunk.js
  41.56 KB (+72 B)       build/static/css/main.94016564.chunk.css
  770 B                  build/static/js/runtime-main.dadc3ad1.js
```

And without the stream player it's
```
  904.81 KB (-1.51 KB)  build/static/js/2.0329ca25.chunk.js
  71.44 KB (-85 B)      build/static/js/main.703e2dfb.chunk.js
  41.56 KB              build/static/css/main.94016564.chunk.css
  770 B                 build/static/js/runtime-main.dadc3ad1.js
```

Reply:
Fair point. Since we're not using some of the fancy features, like preview, I could probably just wrap an img and do away with the import.

Reply:
I can drop antd and just wrap a traditional <img>.

Are we okay leaving `@cloudflare/stream-react` in for videos served from watch.videodelivery.net? Or drop that too. That'll leave us with no new packages.